### PR TITLE
Fix typo

### PR DESCRIPTION
**Pull request by Shine-neko**

### CODE REVIEW

**Comment by snoek09** [comment]
Thanks @Shine-neko :+1: 

Change should be merged to the `2.7` branch as `2.6` is not maintained anymore.

Status: Reviewed

File: reference/forms/types/options/empty_data.rst.inc [hunk 1]
```diff
@@ -22,7 +22,7 @@ selected, you can do it like this::
             'f' => 'Female'
         ),
         'required'    => false,
-        'placeholder' => 'Choose your gender',
+        'attr' => array('placeholder' => 'Choose your gender')
```
Comment (snoek09) [inline reference/forms/types/options/empty_data.rst.inc]: add missing comma at end

**Comment by HeahDude** [comment]
This change is wrong, `placeholder` is a specific option of `ChoiceType`, ref http://symfony.com/doc/current/reference/forms/types/choice.html#placeholder

**Comment by snoek09** [comment]
You're right @HeahDude. I was looking at the wrong type.

**Comment by wouterj** [comment]
Hi @Shine-neko! Can you please explain this change, as [the ChoiceType has a placeholder option](https://github.com/symfony/form/blob/master/Extension/Core/Type/ChoiceType.php#L185)?

Status: works for me

**Comment by xabbuh** [comment]
ping @Shine-neko :)

**Comment by HeahDude** [comment]
This change is wrong, let's close here.

**Comment by javiereguiluz** [comment]
Closing because the proposed change doesn't look OK.

@Shine-neko this time we didn't merge your proposal, but please keep sending new pull reuqests to improve the Symfony docs. Thanks!

**Pull request closed without merging**